docs/oauth_tokens.md

```diff
@@ -100,8 +83,7 @@ This is the feature that requires walking the token set.
 > support anywhere in the codebase to avoid blocking the database.
 
 When we list the tokens for a given service and application, we want to list
-*all* of them, that is, including user-specific tokens. We also support listing
-those that apply to a single user alone.
+*all* of them.
```

Comment:
This sentence makes no sense now.

Reply:
Someone could assume that there's pagination :eyes:

Reply:
The sentence still doesn't make sense. Just delete it or add one about not having pagination.

Reply:
:+1: